docs/pages/includes/database-access/create-user.mdx

```diff
@@ -1,42 +1,22 @@
 Create a local Teleport user with the built-in `access` role:
 
 ```code
-$ tctl users add --roles=access alice
+$ tctl users add \
+  --roles=access \
+  --db-users=\* \
+  --db-names=\* \
+  alice
 ```
 
-The `access` role allows users to see all connected database servers, but
-database names and accounts are restricted to the user's `db_names` and
-`db_users` traits. Normally, these traits come from the identity provider. For
-the local user you've just created you can update them manually to allow it to
-connect to any database as any database user.
+| Flag | Description |
+| ---- | ----------- |
+| `--roles` | List of roles to assign to the user. Builtin `access` role allows them to connect to any database server registered with Teleport. |
+| `--db-users` | List of database usernames the user will be allowed to use when connecting to the databases. Wildcard allows any user. |
+| `--db-names` | List of logical databases (aka schemas) the user will be allowed to connect to within the database servers. Wildcard allows any database. |
```

Comment:
```suggestion
| `--db-names` | List of logical databases (aka schemas) the user will be allowed to connect to within the database servers. A wildcard allows any database. |
```

Reply:
Also, since we haven't specified any database servers before this line, should we say "a database server" instead of "the database servers"?